sdk/python/kfp/cli/cli.py

```diff
@@ -19,6 +19,7 @@
 from .run import run
 from .pipeline import pipeline
 from .diagnose_me_cli import diagnose_me
+from .install import install
 
 @click.group()
 @click.option('--endpoint', help='Endpoint of the KFP API service to connect.')
```

Comment:
Out of the scope of this PR but I think it's beneficial to do a sample in the help message.

```suggestion
@click.option('--endpoint', help='Endpoint of the KFP API service to connect. Example: 1234567890abcdef-dot-us-central1.pipelines.googleusercontent.com')
```

Reply:
do you mean a full command line sample?
like "python3 -m kfp install", right?

Added to welcome message part.

Reply:
What I mean is simply a sample of the endpoint format, as I did in the suggested change, but a full command line sample is great, too.

We've seen quite a few cases where users are confused what should be put into the endpoint section, right?